Source/MMWormhole.h

```diff
@@ -75,7 +77,7 @@
  */
 
 - (instancetype)initWithApplicationGroupIdentifier:(NSString *)identifier
```

Comment:
Yeah as I understand it the NS_ASSUME_NONNULL_BEGIN/END marks everything inside as being nonnull unless tagged otherwise. When I first applied the __nullable without the BEGIN/END macros I started seeing warnings that other points needed annotating.